app/models/account_request.rb

```diff
@@ -73,6 +78,12 @@ def reject!(reason)
     AccountRequestMailer.rejection(account_request_id: id).deliver_later
   end
 
+  # @param reason [String]
+  def close!(reason)
+    return false unless can_be_closed?
+    update!(status: 'admin_closed', rejection_reason: reason)
```

Comment:
Sorry - changing this to `update!` means that this method will never return false - it'll raise an error if it can't save. That means you need to handle this error and return the message to the user rather than rely on true/false returns from this method.

Reply:
I changed, what do you think?